btax/calc_z.py

```diff
@@ -67,11 +67,23 @@ def calc_tax_depr_rates(r, delta, bonus_deprec, deprec_system, tax_methods, fina
 
     z = npv_tax_deprec(tax_deprec_rates, r, tax_methods, financing_list, entity_list)
 
-    # replace tax depreciation rates on land and inventories w/ zero
-    for i in range(r.shape[0]):
-        for j in range(r.shape[1]):
-            z.ix[z['Asset Type']=='Land', 'z'+entity_list[j]+financing_list[i]] = 0.
-            z.ix[z['Asset Type']=='Inventories', 'z'+entity_list[j]+financing_list[i]] = 0.
+    # replace tax depreciation rates on land and inventories w/ zero - unless expense
+    if expense_inventory:
+        for i in range(r.shape[0]):
+            for j in range(r.shape[1]):
+                z.ix[z['Asset Type']=='Inventories', 'z'+entity_list[j]+financing_list[i]] = 1.
```

Comment:
@jdebacker - It's better with `pandas.DataFrame` or `Series` objects to choose to use either:

 * `z.loc` - index based lookup (values within the `.index` attribute to be lookup keys) - what we need here I am guessing, or
 * `z.iloc` - positional based lookup - same as a `numpy.array` integer slicing

The code as written here uses `.ix` which will automatically try to guess whether to use `.loc` or `.iloc`.  Here there does not appear to be ambiguity in using `.ix` as it will default to `.loc`, I think, but in general explicit (`.iloc` or `.loc`) is better than implicit (`.ix`).